.circleci/config.yml

```diff
@@ -92,6 +92,21 @@ commands:
           command: TARGET=<< parameters.target >> cargo test --release --no-default-features --features gpu,blst
           no_output_timeout: 30m
 
+  test_target_blst_gpu_ignored:
+    parameters:
+      target:
+        type: string
+    steps:
+      - *restore-workspace
+      - *restore-cache
+      - run:
+          name: Test blst ignored (GPU) (<< parameters.target >>)
```

Comment:
Should probably take GPU out of the test name (and also above)

Reply:
done